datahub-web-react/src/app/entity/dataset/shared/ExpandingStat.tsx

```diff
@@ -1,18 +1,20 @@
 import React, { ReactNode, useEffect, useRef, useState } from 'react';
 import styled from 'styled-components';
 
-const ExpandingStatContainer = styled.span<{ disabled: boolean; expanded: boolean; width: string }>`
-    overflow: hidden;
-    white-space: nowrap;
-    width: ${(props) => props.width};
+const ExpandingStatContainer = styled.span<{ disabled: boolean; expanded: boolean; width: string; renderCss: boolean }>`
```

Comment:
this seems excessive. 

i'd like to see a solution that only touches this CSS. we should not need new props to solve this problem.

what do you think?

Reply:
As Expanding container for rowcount not working properly in datastasummarynpage .
Added the `renderCss` prop to the ExpandingStat component for a specific use case on the dataStatSummary page for row count . This prop is intended to control the styling of the ExpandingStat component for that page and should not affect the styling on other pages. This addition ensures that other pages remain unaffected and do not break css.